rollup.config.js

```diff
@@ -18,42 +18,49 @@ const tsOpts = {
 };
 
 const globals = {
-    react: 'automat.react',
-    emotion: 'automat.emotion', // TODO remove this dependency
-    '@emotion/core': 'automat.emotionCore',
-    'emotion-theming': 'automat.emotionTheming',
+    react: 'guardian.automat.react',
+    emotion: 'guardian.automat.emotion', // TODO remove this dependency
```

Comment:
Worth checking my PR once it goes live, as I haven't actually implemented this yet. (Although I expect to.)